### PR TITLE
OSDOCS-5088: Added Alibabacloud to install FIPS mode list

### PR DESCRIPTION
**Pull request by dfitzmau**
[OSDOCS-5088](https://issues.redhat.com/browse/OSDOCS-5088)

Version(s):
4.10, 4.11, 4.12 +

Issue:
https://issues.redhat.com/browse/OSDOCS-5088

Link to docs preview: [Installing a cluster in FIPS mode](https://55636--docspreview.netlify.app/openshift-enterprise/latest/installing/installing-fips.html#installation-about-fips-components_installing-fips)

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by dfitzmau** [comment]
Hello, @jianli-wei . You OK to review this PR?

File: installing/installing-fips.adoc [hunk 1]
```diff
@@ -71,6 +71,7 @@ To ensure that containers know that they are running on a host that is using FIP
 To install a cluster in FIPS mode, follow the instructions to install a customized cluster on your preferred infrastructure. Ensure that you set `fips: true` in the `install-config.yaml` file before you deploy your cluster.
 
 * xref:../installing/installing_aws/installing-aws-customizations.adoc#installing-aws-customizations[Amazon Web Services]
+* xref:../installing/installing_alibaba/preparing-to-install-on-alibaba.adoc#preparing-to-install-on-alibaba[Alibaba Cloud]
```
Comment (jianli-wei) [inline installing/installing-fips.adoc:74]: Suggest to be consistent with other providers' links, i.e. use `installing-alibaba-customizations.html` instead. Thanks!

**Review by jianli-wei — CHANGES_REQUESTED** [review]

File: installing/installing-fips.adoc (hunk 1 above)
Comment (dfitzmau) [reply installing/installing-fips.adoc]: Thanks, @jianli-wei , for the feedback. I've updated the link as requested.

**Review by jianli-wei — APPROVED** [review]

**Review by opayne1 — COMMENTED** [review]
/lgtm

**Pull request merged**